recipe/meta.yaml

```diff
@@ -29,8 +29,9 @@ requirements:
     - python
     - pip
     - pybind11
-    - blas     # [not (osx and arm64)]
-    - libblas  # [not (osx and arm64)]
+    - blas       # [not (osx and arm64)]
+    - libblas    # [not (osx and arm64)]
+    - liblapack  # [not (osx and arm64)]
```

Comment:
I needed all of then initially because some files have been in unexpected places. Feel free to make a PR that changes this. CI will tell us then whether we can omit something.